packages/ripple/src/getRippleGlobalNamespace.ts

```diff
@@ -0,0 +1,44 @@
+import { Options } from './utils';
+
+export interface ModuleType {
+  '@leafygreen-ui/ripple': {
+    registeredRippleElements: Map<HTMLElement, Options>;
+    setRippleListener: boolean;
+  };
+}
+
+type SetWindow = Window &
+  typeof globalThis & {
+    __LEAFYGREEN_UTILS__: {
+      modules: {};
+    };
+  };
+
+export type LGWindow = Window &
```

Comment:
No, then we get TS errors: 
```
Element implicitly has an 'any' type because expression of type '"@leafygreen-ui/ripple"' can't be used to index type '{} | ModuleType'.
  Property '@leafygreen-ui/ripple' does not exist on type '{} | ModuleType'.ts(7053)
```

Reply:
Right. Is there a way to do this with less redundancy?

Reply:
I tried, this is the best that I could come up with. I can keep playing with it though